app/views/product-pages/signup/index.html

```diff
@@ -0,0 +1,257 @@
+            ></path>
+          </svg>
+          <span class="govuk-header__logotype-text"> GOV.UK </span>
+        </span>
+        <span class="govuk-header__product-name">
+          Forms
+          <strong class="govuk-tag govuk-phase-banner__content__tag">
+            beta
+          </strong>
+        </span>
+      </a>
+    </div>
+    <div class="govuk-header__content">
+      <nav aria-label="Menu" class="govuk-header__navigation">
+        <button
+          type="button"
+          class="govuk-header__menu-button govuk-js-header-toggle"
+          aria-controls="navigation"
+          aria-label="Show or hide menu"
+          aria-expanded="false"
+        >
+          Menu
+        </button>
+
+        <ul id="navigation" class="govuk-header__navigation-list" hidden="">
+          <li class="govuk-header__navigation-item">
+            <a class="govuk-header__link" href="get-started"> Get started </a>
+          </li>
+          <li class="govuk-header__navigation-item" >
+            <a class="govuk-header__link" href="features"> Features </a>
+          </li>
+          <li class="govuk-header__navigation-item">
+            <a class="govuk-header__link" href="support"> Support </a>
+          </li>
+          <li class="govuk-header__navigation-item" >
+            <a class="govuk-header__link" href="https://admin.research.forms.service.gov.uk/"> Sign in </a>
+          </li>
+        </ul>
+        </ul>
+      </nav>
+    </div>
+  </div>
+</header>
+
+<div class="hero-breadcrumb-banner">
+  <div class="hero hero--breaded">
+    <div class="govuk-breadcrumbs breadcrumbs-hero">
+      <ol class="govuk-breadcrumbs__list">
+        <li class="govuk-breadcrumbs__list-item">
+          <a
+            class="govuk-breadcrumbs__link"
+            href="https://www.gov.uk/service-toolkit#gov-uk-services"
+            >GOV.UK services</a
+          >
+        </li>
+        <li class="govuk-breadcrumbs__list-item">
+          <a
+            class="govuk-breadcrumbs__link"
+            href="#main-content"
+            aria-current="page"
+            >GOV.UK&nbsp;Forms</a
+          >
+        </li>
+      </ol>
+    </div>
+  </div>
+</div>
+{% endblock header %}
+
+{% block main %}
+<div class="masthead">
+  <div class="hero">
+    <div class="hero__content">
+      <div class="hero__body">
+        <h1 class="hero__title">Create online forms for GOV.UK</h1>
+        <div class="hero__inline-image">
+          <img src="images/pay-top-image-5f2b3404.png" alt="Payment screen on mobile and dashboard with refund function." role="presentation">
+        </div>
+        <p class="hero__description">
+          Create an accessible online form in minutes without needing technical knowledge.
+        </p>
+<a href="get-started" role="button" class="govuk-button govuk-button--start button--start_white">Get started</a>        <span class="hero-alternative-action">
```

Comment:
Just a reminder about what we talked about in the catch up earlier. We want to add the chevron back into the button and remove the alternative action to 'register your interest' (at least for the research).